web/admin/src/BigScreen.css

```diff
@@ -0,0 +1,155 @@
+.big-screen {
+  font-size: 3vh;
+  height: 100vh;
+  width: 100vw;
+  background: url(./background.jpg);
+  background-size: cover;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  align-items: center;
+  font: 400 14px -apple-system, BlinkMacSystemFont, 'Fira Sans', 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.bigScreenCell{
+  background-color: white;
+  padding: 20px;
+  margin: 20px 50px;
+  border-radius: 5px;
+}
+
+.cellName {
+  flex-flow: row wrap;
+  flex: 1;
+  display: flex;
+  align-items: center;
+  margin: 0em 0em;
+}
+
+.name {
+  padding: 0 1em;
+}
+
+.big-screen .box {
+  display: block;
+  flex: 1;
+  flex-direction: column;
+  overflow: auto;
+}
+.big-screen .box-content {
+  padding: 2vh 3vw;
```

Comment:
horizontal padding can also be in terms of `vh`.  I would assume the screen width is at least 4/3 of the screen height.